src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -177,6 +179,12 @@ def _get_shell_type():
     else:
         return _remove_cmd_chars(_remove_symbols(os.environ.get('SHELL')))
 
+def _get_error_hash():
+    err = sys.exc_info()[1]
+    return _remove_cmd_chars(_remove_symbols(_get_hash(str(err.message))))
```

Comment:
>message [](start = 63, length = 7)

The use of exception.message is deprecated, no? ([https://www.python.org/dev/peps/pep-0352/](https://www.python.org/dev/peps/pep-0352/)) 

Is there a specific reason to deviate from current guidance (use str(err))?